src/user/room.ts

```diff
@@ -428,23 +428,33 @@ export class Room extends Accessory implements Sayable {
     textOrContactOrFileOrUrl : string | Contact | FileBox | UrlLink,
     mention?                 : Contact | Contact[],
   ): Promise<void> {
+
+    const replyToList: Contact[] = [].concat(mention as any || [])
+
+    const mentionAliasPromiseList = replyToList.length > 0
```

Comment:
I believe `replyToList.length > 0` is not necessary at here.

Rename `mentionAliasPromiseList` to `mentionAliasList` because it's not a promise.

Reply:
mentionAliasPromiseList may contain ```string | null```, mentionAliasList should only contain ```string```. Therefore, I think the line 440 is the one being filter. 
However, you are right. I should give a good name for the one contain ```string | null```